app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -244,69 +218,95 @@ export const Notifications = ({ setHeaderTitleNode }) => {
     }
   };
 
-  const handleDocumentationClick = () => {
-    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.clickDocumentationLink('no_notifications'));
-  };
-
   if (loading) {
     return <SpinningPreloader />;
   }
+  // separate notifications by types
+  const notificationRulesByTypes = Object.groupBy(notifications, (rule) => rule.type);
 
   return (
-    <>
-      {notifications.length ? (
-        <SettingsPageContent>
-          <Layout
-            description={
-              <FormattedDescription
-                content={formatMessage(messages.tabDescription, {
-                  a: (data) => createExternalLink(data, docsReferences.notificationsDocs),
-                })}
-                event={PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.clickDocumentationLink()}
-              />
-            }
-          >
+    <SettingsPageContent>
+      <Layout
+        description={<FormattedDescription content={formatMessage(messages.tabDescription)} />}
+      >
+        <FieldElement
+          className={cx('fieldElement')}
+          withoutProvider
+          description={formatMessage(messages.toggleNote)}
+          dataAutomationId="notificationsEnabledCheckbox"
+        >
+          <div className={cx('toggle')}>
+            <Toggle
+              disabled={isReadOnly}
+              value={enabled}
+              onChange={(e) => toggleNotificationsEnabled(e.target.checked)}
+              dataAutomationId="enabledToggle"
```

Comment:
Discuss all changes connected with `dataAutomationId` with BA and E2E Test Automation (Viktoryia Kuzmitskaya).